hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/main/java/org/apache/hadoop/yarn/server/federation/utils/FederationStateStoreFacade.java

```diff
@@ -728,6 +729,18 @@ public FederationStateStore getStateStore() {
   }
 
   /**
+   * Get Active's SubClusterIds{@link SubClusterId}.
+   *
+   * @return SubClusterId Collection.
+   * @throws YarnException if the call to get active subClusterIds is unsuccessful
+   */
+  public Collection<SubClusterId> getActiveSubClusterIds() throws YarnException {
+    Map<SubClusterId, SubClusterInfo> activeSubClusters =
+        getSubClusters(true);
```

Comment:
Can we put the true in a variable to make it easier to understand what the true represents?

Reply:
Thanks for your suggestion, I will use the `getSubClusters` method.  The way of using parameters (true) is better for code understanding.